src/VisualStudio/Core/Test/CodeModel/CSharp/ExternalCodeClassTests.vb

```diff
@@ -7,6 +7,23 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.CodeModel.CSharp
     Public Class ExternalCodeClassTests
         Inherits AbstractCodeClassTests
 
+#Region "Doc Comment"
+
+        <ConditionalWpfFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.CodeModel)>
+        Public Sub DocComment1()
+            Dim code =
+<Code>
+/// &lt;summary&gt;This is my comment!&lt;/summary&gt;
```

Comment:
Though to be fair, it's pretty much always surrounded with "<member name="Foo:Bar">"